filelog_windows.go

```diff
@@ -1,3 +1,76 @@
 package log4go
 
+import (
+	"archive/zip"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
 const FILELOG_ARCHIVE_REGEX = `\.[0-9]{4}-[0-9]{2}-[0-9]{2}(\.[0-9]{4})?(\.zip)?$`
+
+func (w *FileLogWriter) compressFile(filename string) {
```

Comment:
It seems like a lot of this could be refactored to be platform independent - the file extension and the Writer for the compressed file seem like the extent of the differences?